### PR TITLE
updated makefile and readme with new name for galasactl-darwin-amd64

### PR DESCRIPTION
**Pull request by Wyvinar**
This is now galasactl-darwin-x86_64.

### CODE REVIEW

**Review by techcobweb — APPROVED** [review]

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-144-wcjpr for details. If you are unable to do so, please contact a member of the Galasa team.

File: README.md [hunk 1]
```diff
@@ -353,11 +353,11 @@ Browse the following web site and download whichever built binary files you wish
 The build process builds some docker images with the command-line tools installed.
 This could be useful when wishing to embed a usage of the command-line within a build process which can use a docker image.
 
-- Bleeding edge/Unstable : `docker pull harbor.galasa.dev/galasadev/galasa-cli-amd64:main`
+- Bleeding edge/Unstable : `docker pull harbor.galasa.dev/galasadev/galasa-cli-x86_64:main`
```
Comment (jadecarino) [inline README.md:356]: We are going to leave the Docker images with the same names for now so could you change this back please :)

File: README.md [hunk 2]
```diff
@@ -353,11 +353,11 @@ Browse the following web site and download whichever built binary files you wish
...
 
 ### How to use the docker image
 The docker image has the `galasactl` tool on the path of the docker image when it starts up.
 So, invoke the `galasactl` without installing on your local machine, using the docker image like this:
 ```
-docker run harbor.galasa.dev/galasadev/galasa-cli-amd64:main galasactl --version
+docker run harbor.galasa.dev/galasadev/galasa-cli-x86_64:main galasactl --version
```
Comment (jadecarino) [inline README.md:362]: Same as above

**Review by jadecarino — CHANGES_REQUESTED** [review]

**Review by jadecarino — APPROVED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-144-rqtdd for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by techcobweb — APPROVED** [review]

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-144-vjp67 for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-144-bs29f for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by techcobweb** [comment]
The docker-build phase of the tekton build process is failing.

```
INFO[0000] Retrieving image manifest alpine:3.17.1      
INFO[0000] Retrieving image alpine:3.17.1 from registry [index.docker.io](http://index.docker.io/) 
E0904 09:43:02.794102      16 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
INFO[0002] Built cross stage deps: map[]                
INFO[0002] Retrieving image manifest alpine:3.17.1      
INFO[0002] Returning cached image manifest              
INFO[0002] Executing 0 build triggers                   
INFO[0002] Unpacking rootfs as cmd RUN addgroup galasa &&     adduser -D -G galasa -h /galasa -s /bin/sh galasa requires it. 
INFO[0003] Taking snapshot of full filesystem...        
INFO[0004] ARG platform                                 
INFO[0004] RUN addgroup galasa &&     adduser -D -G galasa -h /galasa -s /bin/sh galasa 
INFO[0004] cmd: /bin/sh                                 
INFO[0004] args: [-c addgroup galasa &&     adduser -D -G galasa -h /galasa -s /bin/sh galasa] 
INFO[0004] Running: [/bin/sh -c addgroup galasa &&     adduser -D -G galasa -h /galasa -s /bin/sh galasa] 
error building image: error building stage: failed to get files used from context: failed to get fileinfo for /workspace/git/repo-cli-pr-144-bs29f/cli/bin/galasactl-linux-amd64: lstat /workspace/git/repo-cli-pr-144-bs29f/cli/bin/galasactl-linux-amd64: no such file or directory
```

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-cli-pr-144-bs29f-r-57vds for details. If you are unable to do so, please contact a member of the Galasa team.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**